Formula/tcping.rb

```diff
@@ -1,7 +1,7 @@
 class Tcping < Formula
   desc "TCP connect to the given IP/port combo"
   homepage "https://web.archive.org/web/20171102193822/linuxco.de/tcping/tcping.html"
-  url "https://mirrors.kernel.org/gentoo/distfiles/tcping-1.3.5.tar.gz"
+  url "https://deb.debian.org/gentoo/distfiles/tcping-1.3.5.tar.gz"
```

Comment:
The same as https://github.com/Homebrew/homebrew-core/pull/39413/files#r280889033.